native/src/components/Link.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react'
```

Comment:
Just a general style issue. If i click on the link, background gets black. Looks a bit ugly ;)

Reply:
can you take a screenshot/video?

Reply:
![Simulator Screen Recording - iPhone 12 Pro Max - 2021-11-09 at 17 42 00](https://user-images.githubusercontent.com/37902063/140966925-f19f2b41-dff2-4173-8463-9fa144d33df3.gif)

Reply:
Could not reproduce this on android emulator o.o

Reply:
Created https://issues.tuerantuer.org/browse/IGAPP-836